src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -299,6 +302,8 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
   @Prop({ default: false })
   readonly validate!: boolean
 
+  @Prop() readonly hasMinimumPartners: boolean
```

Comment:
Whenever possible, you should have a default value for this (or make it required).